script/test

```diff
@@ -125,6 +125,7 @@ do_clean()
 do_check()
 {
     (cd "${OTBR_TOP_BUILDDIR}" \
+        && sudo ninja install \
```

Comment:
May explain why this can fix the test?

Reply:
`ninja install` will install `otbr-agent.conf` to `/etc/dbus-1/system.d`, so it can fix the DBus permission error.

Reply:
Thanks for explanation.